Sources/MapboxDirections/Intersection.swift

```diff
@@ -292,10 +318,21 @@ extension Intersection: Codable {
         if let lanes = try container.decodeIfPresent([Lane].self, forKey: .lanes) {
             approachLanes = lanes.map { $0.indications }
             usableApproachLanes = lanes.indices { $0.isValid }
+            preferredApproachLanes = lanes.indices { ($0.isActive ?? false) }
+            var usableIndications = [LaneIndication]()
+            lanes.forEach { lane in
+                if lane.validIndication != nil {
+                    usableIndications.append(lane.validIndication!)
+                }
+            }
```

Comment:
Nit: This can be written more succinctly and without an implicitly unwrapped optional:

```suggestion
            let usableIndications = lanes.compactMap { lane.validIndication }
```